config.json

```diff
@@ -1727,16 +1727,16 @@
         "groupId": "com.google.code.findbugs",
         "artifactId": "jsr305",
         "version": "3.0.2",
-        "nugetVersion": "3.0.2.6",
+        "nugetVersion": "3.0.2.7",
         "nugetId": "Xamarin.Google.Code.FindBugs.JSR305",
         "dependencyOnly": false,
         "templateSet": "findbugs"
       },
       {
         "groupId": "com.google.dagger",
         "artifactId": "dagger",
-        "version": "2.43.2",
-        "nugetVersion": "2.43.2",
+        "version": "2.44.2",
```

Comment:
This is not just a patch bump, this is a new Java version.

Reply:
I had to manually bump some (remember order of last updates FB was 1st but we merged it as last).
Will fix right now.

Reply:
fixed 2.44.2.1